presentation/common/src/presentation-common/content/Descriptor.ts

```diff
@@ -55,6 +65,26 @@ export namespace SelectClassInfo {
       relatedInstanceClasses: json.relatedInstanceClasses.map((p) => RelatedClassInfo.fromJSON(p)),
     };
   }
+
+  export function decompressSelectClassInfoJSON(compressedSelectClass: CompressedSelectClassInfoJSON, classesMap: { [id: string]: CompressedClassInfoJSON }): SelectClassInfoJSON {
```

Comment:
I see decompression methods defined in two ways:
1. This one is defined under a namespace, named under the thing it decompresses.
2. Some others are defined as private `Descriptor` methods.

We need to choose one or the other, but not both. I'm leaning towards the first approach and naming the methods `fromCompressedJSON`.

Reply:
Renamed all the methods to `fromCompressed...`, also moved Descriptor method `fromCompressedJSON` to be under Descriptor namespace.